VTEX - Orders API (PII version).json

```diff
@@ -3569,49 +3570,156 @@
                 "operationId": "ListOrders2",
                 "parameters": [
                     {
                         "$ref": "#/components/parameters/Content-Type"
                     },
                     {
                         "$ref": "#/components/parameters/Accept"
-                    },
-                    {
-                        "$ref": "#/components/parameters/f_hasInputInvoice"
                     }
                 ],
                 "requestBody": {
                     "content": {
                         "application/json": {
                             "schema": {
                                 "type": "object",
-                                "required": [
-                                    "f_creationDate",
-                                    "per_page",
-                                    "page"
-                                ],
```

Comment:
are these properties really not required?

Reply:
You can even send an empty body and get a `200 OK`, they are really not required.